src/neuroconv/datainterfaces/ecephys/openephys/openephyslegacydatainterface.py

```diff
@@ -0,0 +1,103 @@
+"""Authors: Szonja Weigl, Cody Baker."""
+from typing import Optional
+from warnings import warn
+
+from neo.rawio import OpenEphysRawIO
+from spikeinterface.extractors import OpenEphysLegacyRecordingExtractor
+
```

Comment:
Need to either move these into the local scope of the relevant class methods, or you should be able to remove the `spikeinterface.extractors` import entirely (handled by lazy load mechanism of all extractor-based interfaces)